src/components/ProductMilestoneCloseResultsPage/ProductMilestoneCloseResultsPage.tsx

```diff
@@ -1,5 +1,29 @@
-import { ContentBox } from 'components/ContentBox/ContentBox';
+import { useParams } from 'react-router-dom';
 
-export const ProductMilestoneCloseResultsPage = () => {
-  return <ContentBox padding marginBottom></ContentBox>;
+import { useQueryParamsEffect } from 'hooks/useQueryParamsEffect';
+import { useServiceContainer } from 'hooks/useServiceContainer';
+
+import { ProductMilestoneCloseResultsList } from 'components/ProductMilestoneCloseResultsList/ProductMilestoneCloseResultsList';
+
+import * as productMilestoneApi from 'services/productMilestoneApi';
+
+interface IProductMilestoneCloseResultsPageProps {
+  componentId?: string;
+}
+
+export const ProductMilestoneCloseResultsPage = ({ componentId = 'c1' }: IProductMilestoneCloseResultsPageProps) => {
+  const { productMilestoneId } = useParams();
+
+  const serviceContainerCloseResults = useServiceContainer(productMilestoneApi.getCloseResults);
+  const serviceContainerCloseResultsRunner = serviceContainerCloseResults.run;
+
+  useQueryParamsEffect(
+    ({ requestConfig } = {}) => serviceContainerCloseResultsRunner({ serviceData: { id: productMilestoneId }, requestConfig }),
+    {
+      componentId,
+      mandatoryQueryParams: { pagination: true, sorting: false },
```

Comment:
I would propose to order by close date in descending order, otherwise its hard to get latest results when we have alot of close results.
![image](https://user-images.githubusercontent.com/6870455/227825513-63794cd7-195e-46dc-8d76-32bd1c7d8aa9.png)